include/sound/sof/xman.h

```diff
@@ -100,5 +101,15 @@ struct xman_cc_version {
 	char desc[];	/* null terminated compiler description */
 } __packed;
 
+/* Probe setup */
+struct xman_probe_support {
+	struct xman_elem_header hdr;
+
+	uint32_t probe_points_max;
+	uint32_t injection_dmas_max;
```

Comment:
is this really something the firmware can control? It's to a large extent limited by the number of DMA channels which are NOT allocated by the firmware.

Reply:
Copied from existing code